docs/apm-input-settings.asciidoc

```diff
@@ -220,7 +220,7 @@ Configured values are appended to the default list and used as the value for the
 // tag::rum_response_headers-setting[]
 |
 [id="input-{input-type}-rum_response_headers-setting"]
-`rum_response_headers`
+Custom HTTP response headers
```

Comment:
Maybe worth mentioning that these are in addition to the "Custom HTTP response headers" in the general settings above?